shared/index.native.js

```diff
@@ -22,6 +23,7 @@ class Keybase extends Component {
   componentWillMount () {
     this.store = configureStore()
     setupLocalDebug(this.store)
+    this.store.dispatch(setRouteDef(routeDefs))
```

Comment:
This doesn't currently hot reload. It's kind of tricky due to the RN APIs, because we need to handle it in the `module.hot.accept()` above, which is global, but the store is scoped to component instances. An `EventEmitter` would work, but still feels more complicated than it should be. Move the store up to the top level scope?